ant/__init__.py

```diff
@@ -20,4 +20,6 @@
 # FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
 # DEALINGS IN THE SOFTWARE.
 
+from __future__ import absolute_import, print_function
```

Comment:
Ok, yeah, that is a fair point. Let's keep the imports! About `unicode_literals` is that in regards to the bytearrays as in <http://codereview.stackexchange.com/questions/37959/struct-unpack-on-a-bytearray-that-works-under-python-2-6-6-2-7-and-3-3>?